repos/config/ros2_dashing.lock

```diff
@@ -0,0 +1,520 @@
+#
+#   To update, call `bazel run //repos/config:repos_lock.update` with the right distro set in the WORKSPACE
+#
+repositories:
```

Comment:
@kilian-funk this file reminds me a lot of [the `rosdistro` files here](https://github.com/ros/rosdistro/blob/master/iron/distribution.yaml). I'm not sure if there could be any synergies that we could build in here (e.g. auto-extract doc links from the rosdistro yaml file for packages)...but maybe this is something you've already considered.

Reply:
Yes. Could be that there are some more synergies. I will have a look into this.